Tests/ASTableViewTests.mm

```diff
@@ -608,7 +608,8 @@ - (void)testThatInitialDataLoadHappensInOneShot
 
   // Assert that the beginning of the call pattern is correct.
   // There is currently noise that comes after that we will allow for this test.
-  NSArray *expectedSelectors = @[ NSStringFromSelector(@selector(reloadData)) ];
+  NSArray *expectedSelectors = @[ NSStringFromSelector(@selector(reloadData)),
+                                  NSStringFromSelector(@selector(reloadData)) ];
```

Comment:
Could you update the comments here to explain why there's 2 `reloadData` calls now?